### PR TITLE
Add stubs for glossary

### PR DESCRIPTION
**Pull request by alice-i-cecile**
I used section headings for the definitions to let us link to specific terms.

Feel free to make more requests, or start filling these out! Links are great here.

Start of #151.

### CODE REVIEW

File: design/src/SUMMARY.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # Summary
 
 - [Draft Steam Page](./steam-page.md)
+- [Glossary](./glossary.md)
 - [High Level Analysis](./high-level/index.md)
   - [Game Thesis](./high-level/game-thesis.md)
   - [Genre Analysis](./high-level/genre-analysis.md)
   - [Genre Mechanics](./high-level/genre-mechanics.md)
-  - [Mapping genre mechanics to *Emergence*](high-level/mechanical-translation.md)
+  - [Mapping genre mechanics to *Emergence*](./high-level/mechanical-translation.md)
```
Comment (TimJentzsch) [inline design/src/SUMMARY.md:9]: Wasn't this fixed in the other PR already?

File: design/src/glossary.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# Glossary
+
+These definitions are intended to be relatively brief and opinionated.
+They are not
```
Comment (TimJentzsch) [inline design/src/glossary.md:4]: They are not? :joy:

Comment (alice-i-cecile) [reply design/src/glossary.md:4]:
```suggestion
They are not authoritative!
```

File: design/src/SUMMARY.md (hunk 1 above)
Comment (alice-i-cecile) [reply design/src/SUMMARY.md:9]: This is due to lack of rebasing.

**Pull request merged**